cmd/clusterctl/Dockerfile

```diff
@@ -28,5 +31,18 @@ RUN apk update && apk add ca-certificates openssh-client && rm -rf /var/cache/ap
 
 WORKDIR /root/
 
+RUN apt-get update && apt-get install -y apt-transport-https ca-certificates curl gnupg2 software-properties-common \
```

Comment:
@chenzhiwei Do you think we need to merge all of the `RUN` into one line to reduce the layers of this image?

Reply:
Yes, it is better to merge all commands into a single RUN directive: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers